### ISSUE TITLE
Glare on rectangle shape

### ISSUE THREAD

**Issue by joiel** [mkosir/react-parallax-tilt#16]
### 🐛 Issue/Bug Report Template

**Describe the bug**  
Glare seems not it should
![image](https://user-images.githubusercontent.com/12823424/99213216-9da65700-27ff-11eb-93ee-42e76ff07397.png)

**To Reproduce**  
```
<Tilt
  style={{
    background: "black",
    width: 380,
    height: 180
  }}
  perspective={500}
  glareEnable={true}
  glareMaxOpacity={0.5}
  scale={0.9}
>
    <div>hello</div>
</Tilt>
```

**Expected behavior**  
I guess it shouldn't have a black spot down there. if make it a square glare seem fine.

**Link to demo (required)**  
Please provide demo in online code editor [CodeSandbox](https://codesandbox.io/) or similar.
https://codesandbox.io/s/cranky-haze-9mo54?file=/src/App.js

Issues without a reproduction link are likely to stall.

**Code snippets**  
If applicable, add code samples to help explain your problem.
```
<>
  <Tilt
    style={{
      background: "black",
      width: 380,
      height: 180
    }}
    perspective={500}
    glareEnable={true}
    glareMaxOpacity={1}
    scale={0.9}
  >
    <div
      style={{
        color: "white"
      }}
    >
      RECTANGLE'S GLARE IS SEEM NOT RIGHT
    </div>
  </Tilt>
  <br />
  <Tilt
    style={{
      background: "black ",
      width: 380,
      height: 380
    }}
    perspective={500}
    glareEnable={true}
    glareMaxOpacity={1}
    scale={0.9}
  >
    <div
      style={{
        color: "white"
      }}
    >
      SQUARE'S GLARE IS FINE
    </div>
  </Tilt>
</>
```
**Environment**

- Browser: Chrome
- Version: "react-parallax-tilt": 1.4.66
- ...

**Additional context**

**Comment by lucasfrsi** [comment]
Apparently that's due to the fact of how the glare div size is calculated, but I might be mistaken. I took a quick look at the code and saw that the factor is actually **2**, so the glare div element is twice the size of the wrapper element. If you check the browser dev tools you can check that out too. Besides changing the factor I really don't know what could be done.

**Comment by mkosir** [comment]
Hi @joiel, @lucasfrsi  thanks for providing detailed bug report with reproducible demo link and great PR for bug fix!
Bug fix is merged and deployed in the newest version `1.4.71`, please check and confirm if it's working for you as expected?
